test/selectors.test.js

```diff
@@ -23,6 +23,36 @@ describe("selectors", () => {
     store = createStore(reducer)
   })
 
+  describe("when router not found under 'router' key", () => {
+    beforeEach(() => {
+      const reducer = combineReducers({
+        notTheRouter: connectRouter(history)
+      })
+      store = createStore(reducer)
+    })
+
+    it("throws helpful error", () => {
+      store.dispatch(push('/'))
+      const state = store.getState()
+      expect(() => getLocation(state)).toThrowError('Could not find router reducer in state tree, it must be mounted under "router"')
+    })
+  })
+
+  describe("when something else found under 'router' key", () => {
+    beforeEach(() => {
+      const reducer = combineReducers({
+        router: () => ({ some: 'thing' })
+      })
+      store = createStore(reducer)
+    })
+
+    it("throws helpful error", () => {
+      store.dispatch(push('/'))
+      const state = store.getState()
+      expect(() => getLocation(state)).toThrowError('ould not find router reducer in state tree, it must be mounted under "router"')
```

Comment:
nit: typo. `Could`. Hmm..why did the test pass?

Reply:
Found that Jest uses RegExp to test. So, if we pass a string, it will check if the input contains this specified substring or not (https://github.com/facebook/jest/blob/master/packages/expect/src/toThrowMatchers.js#L54).

So, let's use 
```js
toThrowError(/^Could not find router reducer in state tree, it must be mounted under "router"$/)
```
for both test cases.

Reply:
Fixed